pip/requirements.txt

```diff
@@ -258,8 +259,8 @@ tables==3.6.1 ; python_version>'3.0'
 #tensorboard: Need separate packages for py2 and py3 to properly get the correct sources
 tensorboard==2.0.0 ; python_version<'3.0'
 tensorboard==2.0.0 ; python_version>'3.0'
-tensorflow==2.1.2 ; python_version<'3.0'
-tensorflow==2.1.2 ; python_version>'3.0'
+tensorflow==2.3.1 ; python_version<'3.0'
```

Comment:
the tensor flow docs say python<3 is not supported...

Reply:
should we consider removing it now ?  we build this wheel, if it builds (and run) we can have it

Reply:
@davidlange6 , it still builds for py2 :-)

Reply:
@mrodozov , although we build our own sheel but we still use this way to build py*-tensorflow packages. This allows us to check the missing dependencies